cfpb/cfgov-refresh
# Adds accessible tagline atom

Fixes [GHE]/CFGOV/platform/issues/2610

Adds new accessible tagline atom that's shared between the header and footer.

## Additions

- Adds tagline atom.

## Changes

- Lowercases "government" in the tagline.

## Testing

1. Visit the homepage.
2. Resize to small screen size.
3. Check wrapping of tagline inside the hamburger menu at various sizes. It should generally lay out with US government all on one line.
4. Close hamburger menu.
3. Press command + f5 to enter VoiceOver.
4. Scroll to the footer.
5. click somewhere in the footer and press tab key.
5. Press ctrl + command + right arrow till you get to the tagline in the footer and verify it reads "An official website of the United States government"
